app/src/main/java/com/sunasterisk/weather/screen/weather/WeatherFragment.kt

```diff
@@ -36,23 +52,39 @@ class WeatherFragment : Fragment(), WeatherContract.View {
     }
 
     private fun initView() {
-       // TODO RECYCLERVIEW HOURLY, DAILY
+        // Hourly RecyclerView
+        recyclerViewHourly.setHasFixedSize(true)
+        recyclerViewHourly.adapter = hourlyAdapter
+        // Daily RecyclerView
+        recyclerViewDaily.setHasFixedSize(true)
+        recyclerViewDaily.adapter = dailyAdapter
     }
 
     private fun initData() {
-        val argument = arguments
         presenter.setView(this)
+        val argument = this.arguments
         argument?.let {
-           presenter.getWeather(it.getDouble(WeatherEntry.LATITUDE),
-                                it.getDouble(WeatherEntry.LONGITUDE))
+           presenter.getWeather(it.getDouble(Constant.LATITUDE_KEY),
+                                it.getDouble(Constant.LONGITUDE_KEY))
```

Comment:
```suggestion
           presenter.getWeather(
               it.getDouble(Constant.LATITUDE_KEY),
               t.getDouble(Constant.LONGITUDE_KEY)
           )
```
https://kotlinlang.org/docs/reference/coding-conventions.html#method-call-formatting